ncolony/tests/helper.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Moshe Zadka
+# See LICENSE for details.
+"""Helper functions"""
+import json
+
+def dumps2utf8(something):
```

Comment:
If this is worth abstracting into a function for tests, it's worth using elsewhere.  Also, you should write a test for this!

Reply:
#50